src/Telemetry/readme.md

```diff
@@ -0,0 +1,36 @@
+
+Install:
+https://github.com/microsoft/artifacts-credprovider
+
+
+Run:
+iex "& { $(irm https://aka.ms/install-artifacts-credprovider.ps1) }"
+
+Run:
+dotnet restore --interactive   
+
+Env vars:
```

Comment:
this is a bit terse, could you explain when/how each one is used.

Reply:
I've included a comprehensive readme last night